modcc/expression.cpp

```diff
@@ -324,8 +337,13 @@ expression_ptr StoichTermExpression::clone() const {
 }
 
 void StoichTermExpression::semantic(scope_ptr scp) {
+    error_ = false;
     scope_ = scp;
+
     ident()->semantic(scp);
+    if(ident()->has_error()) {
```

Comment:
Similar comment as above: can you use the error string that was set when calling `ident()->semantic(scp)`?

Reply:
Yes, I can just set the error to true

Reply:
We can copy the error of `ident` into the error string of the `StoichTermExpression`. I think that would be a more helpful message. I'll fix that.